test/spec/vis/vis.spec.js

```diff
@@ -974,6 +975,36 @@ describe('vis/vis', function () {
       this.vis = new Vis();
     });
 
+    describe('remove dataview', function () {
```

Comment:
'remove dataview' -> 'when a dataview is removed'. As I said in my previous comment, I'd also add a test here: `when a dataview is added`.